zerver/views/events_register.py

```diff
@@ -47,6 +47,7 @@ def events_register_backend(
         "client_gravatar", default=None, json_validator=check_bool
     ),
     slim_presence: bool = REQ(default=False, json_validator=check_bool),
+    presence_last_update_id: Optional[int] = REQ(default=None, json_validator=check_int),
```

Comment:
Hmm yeah the idea was in case the mobile app wanted to use this; and also in case we were changing presence events format based on whether the client is `last_update_id`-based. But we're not changing events right now, so no point perhaps. Removed this for now.

Reply:
I think a feature to have mobile be able to just used cached data on this detail is probably a good idea, so worth bringing it up in API design.

Reply:
https://chat.zulip.org/#narrow/stream/378-api-design/topic/presence.20rewrite.20.3E.20add.20last_update_to.20.2Fregister.3F